backend/src/modules/slide-objects/slide-objects.service.ts

```diff
@@ -8,82 +8,135 @@ import {
 } from 'src/schemas/slide-object.schema';
 import { UsersService } from '../users/users.service';
 import { SlidesService } from '../slides/slides.service';
+import { SlideObjectResponseObject } from '../../shared/interfaces/response-objects/SlideObjectResponseObject';
 
 @Injectable()
 export class SlideObjectsService {
+  private readonly pageSize = 3;
   constructor(
     @InjectModel(SlideObject.name)
     private readonly slideObjectModel: Model<SlideObject>,
     private readonly userService: UsersService,
     private readonly slideService: SlidesService,
   ) {}
 
-  async findAll(): Promise<SlideObjectDocument[]> {
-    return this.slideObjectModel.find().exec();
+  async getSlideObjects(
```

Comment:
What is the use case of this method?

Reply:
To remain consistency - as for the rest services there are methods called this way (its started out when method findUserById was used by auth service and users controller - i decided to divide the method into two - first one returning the whole document and the second one returning only response object. For the future reference, we might need it as well when operating on slide.service

Reply:
I don't see the use case for fetching objects from all slides of all boards. I don't think we want to have code that won't be used. However I can live with that so I approve.